client/state/atomic/transfers/actions.ts

```diff
@@ -15,7 +15,13 @@ export interface AtomicTransfer {
 	is_stuck: boolean;
 	is_stuck_reset: boolean;
 	in_lossless_revert: boolean;
-	error?: string;
+}
+
+export interface AtomicTransferError {
+	name: string; // "NotFoundError"
+	status: number; // 404
+	message: string; // "Transfer not found"
+	code: string; // "no_transfer_record"
```

Comment:
<3

Reply:
This is the same as the AtomicSoftwareError type. Should we combine them to a AtomicSetupError or something?

Reply:
I'm not sure we can guarantee they will keep equal?

Reply:
They're actually `WPError from 'wp-error'` but we don't have the package installed.